packages/material-ui/src/RadioGroup/RadioGroup.test.js

```diff
@@ -37,6 +37,20 @@ describe('<RadioGroup />', () => {
     assert.strictEqual(handleKeyDown.args[0][0], event);
   });
 
+  it('should support uncontrolled mode', () => {
+    const wrapper = shallow(
+      <RadioGroup name="group">
+        <Radio value="one" />
+        <Radio value="second" />
+      </RadioGroup>,
+    );
+
+    const firstRadioGroupChild = wrapper.children().first();
+    const event = { target: { value: 'one' } };
+    firstRadioGroupChild.simulate('change', event, true);
+    assert.strictEqual(wrapper.instance().state.value, 'one');
```

Comment:
Never test internal state. It might change over time without actually breaking anything. Rather find the `input` that should be checked and assert that it is checked
```js
    assert.strictEqual(wrapper.find('input[value="one"]["checked"]').exists(), true);
```
Not sure if this is how it works. If not a less optimal (in the sense of accessing internals) solution could be
```js
    assert.strictEqual(wrapper.find('input[value="one"]').props().checked, true);
```